model/project_ref_test.go

```diff
@@ -427,3 +428,44 @@ func TestAddPermissions(t *testing.T) {
 	assert.NoError(err)
 	assert.Contains(dbUser.Roles(), "admin_project_myProject")
 }
+
+func TestUpdateAdminRoles(t *testing.T) {
+	expect := assert.New(t)
```

Comment:
avoid declaring expect and just pass t to assert functions.

Also you probably want to use require here.

Reply:
done